### PR TITLE
feat: turnaround mapping

### PR DESCRIPTION
**Pull request by MattPDengler**
adding a turnaround mapping to starter kit

### CODE REVIEW

File: mappings-examples/jedi-850-to-jedi-855/mapping.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "name": "jedi-850-to-jedi-855",
```
Comment (mthenw) [inline mappings-examples/jedi-850-to-jedi-855/mapping.json:2]:
```suggestion
  "name": "JEDI 850 to JEDI 855",
```

Comment (asennikov) [reply mappings-examples/jedi-850-to-jedi-855/mapping.json:2]: if we do that - I'd recommend updating the rest of the examples for consistency too:

https://github.com/Stedi/starter-kit/blob/main/mappings-examples/jedi-846-shopify-inventorylevel/mapping.json#L2

Comment (MattPDengler) [reply mappings-examples/jedi-850-to-jedi-855/mapping.json:2]: All are currently lowercase so I'm inclined to leave it.

**Comment by MattPDengler** [comment]
@mthenw @asennikov can one of you approve?

File: mappings-examples/jedi-850-to-jedi-855/mapping.json (hunk 1 above)
Comment (asennikov) [reply mappings-examples/jedi-850-to-jedi-855/mapping.json:2]: well, not all of them - I think we have all the different flavors:

- whitespace-separated sentence: https://github.com/Stedi/starter-kit/blob/3cc8731ec48ffce4b092145cee52acfa553698d2/mappings-examples/jedi-846-to-csv/mapping.json#L2
- mixed-case with dashes: https://github.com/Stedi/starter-kit/blob/3cc8731ec48ffce4b092145cee52acfa553698d2/mappings-examples/jedi-214-shopify-fulfillment/mapping.json#L2
- lowercase with dashes (the most common one): https://github.com/Stedi/starter-kit/blob/3cc8731ec48ffce4b092145cee52acfa553698d2/mappings-examples/amazon-jedi-810-to-quickbooks-online-supplier-bill/mapping.json#L2

**Pull request merged**